packages/cardano-services/test/util.ts

```diff
@@ -85,3 +86,30 @@ export const clearDbPools = async ({ main, healthCheck }: DbPools) => {
   await main?.end();
   await healthCheck?.end();
 };
+
+export const projectorConnectionConfig = {
+  database: process.env.POSTGRES_DB_PROJECTION!,
+  host: process.env.POSTGRES_HOST!,
+  password: process.env.POSTGRES_PASSWORD!,
+  port: Number.parseInt(process.env.POSTGRES_PORT!),
+  username: process.env.POSTGRES_USER!
+};
+export const projectorConnectionConfig$ = of(projectorConnectionConfig);
+
+// export const initializeDataSource = async (props: Pick<CreateDataSourceProps, 'entities' | 'extensions'>) => {
```

Comment:
Do we want to keep this commented code?

Reply:
No it's trash, e70b23b8f34004aa600ecc9427c854bf8401c4a0